simple_metering.cpp

```diff
@@ -93,6 +93,7 @@ void DeRestPluginPrivate::handleSimpleMeteringClusterIndication(const deCONZ::Ap
                     consumption = static_cast<quint64>(round((double)consumption / 10.0)); // 0.1 Wh -> Wh
                 }
                 else if (modelId == QLatin1String("SP 120") ||                    // innr
+                         modelId == QLatin1String("SP 234") ||                    // innr
```

Comment:
I am not sure this is reporting correctly, what units is consumption supposed to be in, Wh?

Right now the outlet is reports consumption at 0.29 kWh and I havent really used it yet, that seems like more than I would expect for just powering the radio...?

Reply:
I will use it properly this weekend and get a better feel, I can also plug a lamp + incandescent in for an hour and get the real answer later this weekend to get a more concrete answer other than a "feeling"